spec.bs

```diff
@@ -5922,8 +6001,15 @@ dictionary ReportingBrowserSignals {
   <dd>Copied from [=leading bid info/component seller=]
   <dt>{{ReportingBrowserSignals/buyerAndSellerReportingId}}
   <dd>Set if the winning ad had a [=interest group ad/buyer and seller reporting ID=] set in its
-    listing in the interest group, and that value was
-    [=query reporting ID k-anonymity count|jointly k-anonymous=] combined with interest group owner,
+      listing in the interest group, and that value was
+      [=query reporting ID k-anonymity count|jointly k-anonymous=] combined with interest group owner,
+      bidding script URL, [=ad creative=] URL, and
+      [=generated bid/selected buyer and seller reporting ID=] if it [=map/exists=], otherwise null.

```

Comment:
is the "otherwise null" for "if it [=map/exists=]?
If so, [=generated bid/selected buyer and seller reporting ID=] is null or a string, so it already covers both cases.